dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/Application.cs

```diff
@@ -526,6 +530,58 @@ public AI<TState> AI
             return this;
         }
 
+        /// <summary>
+        /// Handles config fetch events.
+        /// </summary>
+        /// <param name="handler">Function to call when the event is triggered.</param>
+        /// <returns>The application instance for chaining purposes.</returns>
+        public Application<TState, TTurnStateManager> OnConfigFetch(ConfigHandler<TState> handler)
+        {
+            Verify.ParamNotNull(handler);
+            RouteSelector routeSelector = (turnContext, cancellationToken) => Task.FromResult(
+                string.Equals(turnContext.Activity.Type, ActivityTypes.Invoke, StringComparison.OrdinalIgnoreCase)
+                && string.Equals(turnContext.Activity.Name, CONFIG_FETCH_NAME));
```

Comment:
Need to verify channel is msteams since this is a Teams-specific event.

Reply:
Thanks, updated. Just curious why bot builder does not verify msteams channel for config/fetch and config/submit.

Reply:
I think that's an oversight.